Make Title easily legible when a tab is selected

rwmpelstilzchen opened the pull request:
`hue_2` (bg) and `hue_4` (fg) don’t mix well neither in `background=dark` nor in `background=light`.

Current situation (selected; look how unpleasant the `2` at the top-left corner looks):
![one-orig-selected](https://cloud.githubusercontent.com/assets/3979978/21772788/51410b22-d695-11e6-886c-a89970577e21.png)

Current situation (not selected):
![one-orig-notselected](https://cloud.githubusercontent.com/assets/3979978/21772766/424b7ec2-d695-11e6-811e-3cd99fee53e9.png)

Proposed solution (selected):
![one-new-selected](https://cloud.githubusercontent.com/assets/3979978/21772769/43227f9e-d695-11e6-8f27-83786632110e.png)

Proposed solution (not selected):
![one-new-notselected](https://cloud.githubusercontent.com/assets/3979978/21772767/42ca635e-d695-11e6-949f-37a90ca4dc71.png)

rakr commented:
Thanks for the PR,

I've never noticed this glitch. However because your solution implies modifying the `Title` highlight, this has some impact on other languages (like HTML).
The issue that I have is that as I'm using Neovim in terminal only I don't get to see the number that appears at the left of your tabline, I would suggest we make some experiments you and I and settle with the best solution/colormix.


Current version - light
![original](https://cloud.githubusercontent.com/assets/110558/21773925/e1b82876-d6e4-11e6-9a36-6d7c312d7193.png)

Proposed solution - light
![proposed-solution](https://cloud.githubusercontent.com/assets/110558/21773932/e9bed452-d6e4-11e6-9aeb-e78c20115764.png)

Current solution - dark
![original-dark](https://cloud.githubusercontent.com/assets/110558/21773938/edcb06c4-d6e4-11e6-8b65-cf210f4dcea0.png)

Proposed solution - dark
![proposed-solution-dark](https://cloud.githubusercontent.com/assets/110558/21773946/f394c888-d6e4-11e6-9822-4e19114b0e8f.png)

rwmpelstilzchen commented:
Thanks.

In order for the number to appear, split the window (`:sp`). The number indicates the number of windows in a tab. I use Neovim in terminal ([st](http://st.suckless.org)) as well.

I’ve added three commits. The first one (186027a) links `htmlH1` and `manTitle` to `String` (also `hue_4`; any better idea?). These are the only ones linking to Title AFAIK. The other commits improve `man` highlighting (f31fb49) and colorscheme styling (a158c71).

rakr commented:
thanks for the PR and sorry for the delay

rwmpelstilzchen commented:
Thanks :-)
